src/presentations/indexable-post-type-presentation.php

```diff
@@ -70,6 +86,43 @@ public function generate_og_type() {
 		return 'article';
 	}
 
+	/**
+	 * Generates the open graph article author.
+	 *
+	 * @return string The open graph article author.
+	 */
+	public function generate_og_article_author() {
+		$post = $this->post->get_post( $this->model->object_id );
```

Comment:
You can call `$this->replace_vars_object` here. Under the hood it will call `generate_replace_vars_object` and that will make sure the value is cached.